spec/unit/yardstick/parser/class_methods/parse_string_spec.rb

```diff
@@ -5,33 +5,42 @@
 describe Yardstick::Parser, '.parse_string' do
   subject(:document_set) { described_class.parse_string(string) }
 
-  let(:string)        { double('string', to_str: 'body')                      }
-  let(:method_object) { double(file: 'foo.rb', line: 4, docstring: docstring) }
-  let(:docstring)     { double('docstring')                                   }
-
-  let(:method_objects) { [method_object] }
+  let(:string)         { double('string', to_str: 'body')                      }
+  let(:method_object1) { double(file: 'bar.rb', line: 2, docstring: 'barL2')   }
+  let(:method_object2) { double(file: 'foo.rb', line: 1, docstring: 'fooL1')   }
+  let(:method_object3) { double(file: 'foo.rb', line: 3, docstring: 'fooL3')   }
+  let(:method_objects) { [method_object3, method_object1, method_object2]      }
 
   before do
     expect(YARD).to receive(:parse_string).with('body')
-    allow(YARD::Registry).to receive(:all).with(:method).and_return(method_objects)
+    expect(YARD::Registry).to receive(:clear)
+    allow(YARD::Registry)
+      .to receive(:all).with(:method).and_return(method_objects)
   end
 
   it { should be_a(Yardstick::DocumentSet) }
 
-  its(:length) { should be(1) }
+  its(:length) { should be(3) }
+
+  context 'ordering' do
+    subject { document_set.map(&:docstring) }
+    it { should eql(%w(barL2 fooL1 fooL3)) }
```

Comment:
This is a small preference but I prefer to use `%w[` and `]`. I prefer square brackets because I like how it resembles the normal `[` and `]` and reminds me I am working with an Array.